packages/integration-tests/src/test-project-setup/data_storage_auth_with_triggers.ts

```diff
@@ -460,4 +445,42 @@ class DataStorageAuthWithTriggerTestProject extends TestProjectBase {
       throw err;
     }
   };
+
+  private assertScheduleInvokesFunction = async () => {
+    const TIMEOUT_MS = 1000 * 60 * 2; // 2 minutes
+    const startTime = Date.now();
+    let messageCount = 0;
+
+    const getQueueUrlResponse = await this.sqsClient.send(
+      new GetQueueUrlCommand({
+        QueueName: 'amplify-testFuncQueue',
+      })
+    );
```

Comment:
Will this not create race conditions (or failures on trying to create multiple queues with the same name) between multiple text executions at the same time? Can we create a unique SQS queue every single time and find it in the stack using `findByBackendIdentifier`

Reply:
Done in https://github.com/aws-amplify/amplify-backend/pull/1778/commits/bdc807109ba34b5717c5cbd98b73bf9ac5061e48